utils/validate_hosts_for_delete.py

```diff
@@ -1,18 +1,19 @@
 from app import create_app
-from app import events
 from app import UNKNOWN_REQUEST_ID_VALUE
 from app.environment import RuntimeEnvironment
-from app.events import HostEvent
 from app.logging import get_logger
 from app.logging import threadctx
 from app.models import Host
+from app.queue.events import build_event
+from app.queue.events import EventType
+from app.queue.events import HostEvent
 
 logger = get_logger("utils")
 
 
 def test_validations(host):
     schema = HostEvent()
-    event = events.delete(host)
+    event = build_event(EventType.delete, host)
     deserialized = schema.loads(event)
     return deserialized.errors
```

Comment:
This is all handled by [_build_event_](https://github.com/RedHatInsights/insights-host-inventory/blob/294e1760096cf97139c64449bf23d0bc4704064d/app/queue/events.py#L107) now. Moreover it uses strict validation that raises a ValidationError on failure.

```suggestion
    try:
        build_event(EventType.delete, host)
    except ValidationError as error:
        return error.messages
    else:
        return {}
``